Tests/Marketplace/upload_packs.py

```diff
@@ -374,7 +374,9 @@ def get_private_packs(private_index_path):
             if metadata:
                 private_packs.append({
                     'id': metadata.get('id'),
-                    'price': metadata.get('price')
+                    'price': metadata.get('price'),
+                    'vendorId': metadata.get('vendorId'),
+                    'vendorName': metadata.get('vendorName')
```

Comment:
None is fine or do you need to check that first?
```suggestion
                    'vendorName': metadata.get('vendorName'),
```